auth_brute_force/controllers/main.py

```diff
@@ -32,7 +32,9 @@ def web_login(self, redirect=None, **kw):
                 # Test if remote user is banned
                 banned = banned_remote_obj.search([('remote', '=', remote)])
                 if banned:
-                    request.params['password'] = ''
```

Comment:
Maybe I'm wrong, but it you make this sentence, conditionable, you'll change the algorithm. you should write better : 

```
password = request.params['password'] 
request.params['password'] = ''
```
and later
```
attempt_id.write({'password': password})
```

Reply:
It wouldn't but I've changed it anyway.